src/components/UI/atoms/Image/Image.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { borderRadius } from '<variables>/border';
+import { boxShadows } from '<variables>/shadows';
+import { borderColors } from '<variables>/colorPalette';
+
+/**
+ * @description - Image Component
+ *
+ * @prop {string} width - image width
+ * @prop {string} height - image height
+ * @prop {string} ImageUrl - image source
+ * @prop {string} altText - alt text
+ * @prop {string} boxShadow - shadowType
+ * @prop {boolean} borderColor- borderColor
+ * @prop {string} borderRadiusType - borderRadiusType
+ * @return {component} Image
+ */
+
+const Image = (
+  {
+    ImageUrl, altText, width, height, boxShadow, borderRadiusType, borderColor,
+  },
+) => {
+  return <Image.Container
+    src={ImageUrl}
```

Comment:
change to `imageUrl`

Reply:
I think it would make sense to define a default placeholder here if no imageUrl is defined